pkgs/top-level/all-packages.nix

```
@@ -5154,6 +5154,9 @@ in
 
   libhandy = callPackage ../development/libraries/libhandy { };
 
+  # Needed for apps that still depend on the unstable verison of the library (not libhandy-1)
+  libhandy_0 = callPackage ../development/libraries/libhandy/0.x.nix { };
```

I would do something like this, and then remove all of the fixup commits
```suggestion
  libhandy = libhandy_0;
  # stable branch
  libhandy_1 = callPackage ../development/libraries/libhandy { };
  # unstable verison of the library (not libhandy-1)
  libhandy_0 = callPackage ../development/libraries/libhandy/0.x.nix { };
```